tests/e2e/framework/rawVM.go

```diff
@@ -135,8 +149,11 @@ func (vm *GCPRawVM) Teardown() error {
 			return err
 		}
 	}
-	_, err := u.Shell(vm.baseCommand("delete"))
-	return err
+	if !vm.UseMason {
```

Comment:
I am guessing if mason is used we rely on session timeout to release the VM? If the lease is short we should be fine, but it is always nice if we could release the VM timely and explicitly.

Reply:
the VM + cluster will be released once the prow script exits.

Reply:
but I noticed that I can skip the vm deletion in the tear down